src/configuration.rs

```diff
@@ -34,29 +33,43 @@ pub struct Configuration {
     pub metrics: Metrics,
 }
 
-pub(super) fn setup_configuration(args: Args) -> Configuration {
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+pub struct Args {
```

Comment:
Doesn't this already exist somewhere else. Ie the fields inside here?

Reply:
It looks like it was moved here from main.rs.